sycl/plugins/opencl/pi_opencl.hpp

```diff
@@ -18,9 +18,12 @@
 #define PI_OPENCL_HPP
 
 #include <climits>
+#include <map>
+#include <pi2ur.hpp>
```

Comment:
This dependency of Unified Runtime on OpenCL path is unexpected and I think is wrong.
What exactly is it using from there?

Reply:
The plan is to move this OpenCL Plugin under the hood of Unified Runtime and make it the OpenCL Adapter of the later.  And then it will be able to use utilities from UR, which you are presumably doing here. But you create a counter-dependency, seems to me.

Reply:
Hi Sergey,

Thanks for feedback. If you are referring to inclusion to pi2ur.hpp, I am using the ReturnHelper data structure from pi2ur.hpp. This is something done in L0 plugin as well. I can easily replace this by copying the ReturnHelper data structure, but I did not want to reinvent the wheel. 
Can you please elaborate on the counter-dependency?

Thanks again.

Reply:
My thinking was that we were to move this plugin to UR, not UR to this plugin, which is what I meant by "counter-dependency". On a second thought it is probably OK as is since it starting to use UR already, which will be available to it when moved to UR umbrella. I addressed issues that I was encountering in 

https://github.com/intel/llvm/pull/8088/commits/487f982589c2febecfa6ec4a3bc571ebc1fdf037

So - never mind.